tasks/users.yml

```diff
@@ -15,6 +20,8 @@
     name: "{{ item.key }}"
     shell: "{{ item.value.shell|default('/bin/bash') }}"
     password: "{{ item.value.passwd|default('$6$mDFWEb5pDY$C9ZTuNjTTSyh0uIBoZALAV6isFY4dO8gBN2/xJ0yX2rejvr2wKp/wMmHwvoC.gD8NaeozxjhWvNHp3rJEJdJj1') }}"
+    append: true
+    groups: "ssh-users"
```

Comment:
bitte in die bestehende liste aufnehmen und append wieder raus nehmen oder auf no setzen, wir wollen genau wissen, in welchen gruppen die admin user sind und diese statisch setzen.